client/termsetting/handlers/geneVariant.ts

```diff
@@ -83,9 +89,9 @@ export function fillTW(tw: GeneVariantTW, vocabApi: VocabApi, defaultQ = null) {
 
 	// groupsetting
 	// fill term.groupsetting
-	if (!tw.term.groupsetting) tw.term.groupsetting = geneVariantTermGroupsetting
+	if (!tw.term.groupsetting) (tw.term.groupsetting as any) = geneVariantTermGroupsetting
 	// fill q.groupsetting
-	if (!tw.q.groupsetting) tw.q.groupsetting = {}
+	if (!tw.q.groupsetting) (tw.q.groupsetting as any) = {}
```

Comment:
Can `tw.q.groupsetting` be typed `tw.q.groupsetting as GroupSetting` just one time? Then no need to to declare the type of `tw.q.groupsetting` or `self.q.groupsetting` in the rest of the code.

Reply:
> Can `tw.q.groupsetting` be typed `tw.q.groupsetting as GroupSetting` here? Then no need to to declare the type of `tw.q.groupsetting` or `self.q.groupsetting` in the rest of the code.

My IDE throws type errors when I did that. If you see something different, please share.